contracts/package.json

```
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "compile": "hardhat starknet-compile",
+    "compile:l1": "npx hardhat compile",
```

Why npx if we have it as a dependency?

```suggestion
    "compile:l1": "hardhat compile",
```

---

It seems to be the way hardhat scripts are executed everywhere